services/QuillLMS/spec/models/current_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
I don't think we need to remove this file, but as far as I can tell what we're really testing here is stuff that should be covered in `ActiveSupport::CurrentAttributes`.  None of the code in your model definition has any implications for this spec (like `reset` isn't even in your code).  So, in theory, this behavior should be tested by the underlying parent class.

That said, this spec file serves as a useful bit of documentation.  For instance, I didn't know that `reset` was a call that could be made, so seeing it here was helpful.